### PR TITLE
Minor Update Assortment - NAWI Update, Updated for Python 3 + Added ability to pull EarningsRecord from SSA XML 

### PR DESCRIPTION
**Pull request by kfowlks**
Fixed issues with Print function on Python 3
Updated the NAWI tables for 2016 to 2019 years
Added ability to read prior earning data from a SSA downloaded file if present in the same directory.

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (antkowiak) [inline .idea/.gitignore]: The top level .gitignore is ok to check in. But maybe better not to include another one in an IDE-specific subdirectory like ".idea"

File: social_security.py [hunk 2]
```diff
@@ -21,13 +21,16 @@
 # Import modules
 from datetime import datetime
 from math import floor
+import xml.etree.ElementTree as et
+#from lxml import etree
```
Comment (antkowiak) [inline social_security.py]: Commented out line should be removed

File: social_security.py [hunk 3]
```diff
@@ -51,6 +54,17 @@
     2017 :      0.0
 }
 
+try:
+    namespaces = {'osss': 'http://ssa.gov/osss/schemas/1.0'}
```
Comment (antkowiak) [inline social_security.py:57]: I like your idea to be able to parse out the values out of XML.
Perhaps an improvement would be to check if a command line parameter has been passed to the script, and use that as the filename to read from?

File: social_security.py [hunk 4]
```diff
@@ -160,12 +175,13 @@ def KeyWithMaxVal(d):
...
 # Print the results
-print "Top 35 Years of Adjusted Earnings _________", "{:11.2f}".format(Top35YearsEarnings)
-print "Average Indexed Monthly Earnings (AIME) ___", "{:11.2f}".format(AIME)
-print "First Bend Point __________________________", "{:11.2f}".format(FirstBendPoint)
-print "Second Bend Point _________________________", "{:11.2f}".format(SecondBendPoint)
-print "Normal Monthly Benefit ____________________", "{:11.2f}".format(NormalMonthlyBenefit)
-print "Normal Annual Benefit _____________________", "{:11.2f}".format(NormalMonthlyBenefit * 12.0)
-print "Reduced (70%) Monthly Benefit _____________", "{:11.2f}".format(ReducedMonthlyBenefit)
-print "Reduced (70%) Annual Benefit ______________", "{:11.2f}".format(ReducedMonthlyBenefit * 12.0)
+print ("Top 35 Years of Adjusted Earnings _________", "{:11.2f}".format(Top35YearsEarnings))
```
Comment (antkowiak) [inline social_security.py]: In order to make all of these prints look good with both Python2 and Python3, they should all be changed to use the plus operator for concatenation, like:

print ("Top 35 Years of Adjusted Earnings _________" + "{:11.2f}".format(Top35YearsEarnings))

**Review by antkowiak — CHANGES_REQUESTED** [review]
Thanks for suggesting some improvements to the script!
I've made some comments for a few further improvements/cleanups before I merge your PR.

File: .idea/.gitignore (hunk 1 above)
Comment (kfowlks) [reply .idea/.gitignore]: Agreed

File: social_security.py (hunk 2 above)
Comment (kfowlks) [reply social_security.py]: Agreed

File: social_security.py (hunk 3 above)
Comment (kfowlks) [reply social_security.py:57]: I purposely left that out as to not add complexity to such a simple program. The intent was to use a default convention that would work if present. While allowing for the script to still function as you had it with hand populated values if desired.

**Pull request merged**